Resources/Prototypes/_LostParadise/Entities/Clothing/Uniforms/jumpsuitsnpz.yml

```
@@ -41,4 +41,26 @@
+  - type: Sprite
+    sprite: _LostParadise/Clothing/Uniforms/NPZ/representive.rsi
+  - type: Clothing
+    sprite: _LostParadise/Clothing/Uniforms/NPZ/representive.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: LPPClothingUniformJumpsuitCamoNPZ
+  name: NPZ urban camo
+  description: No, this is not a reference to some fanatics from Chernobyl.
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)